pkg/version/version.go

```diff
@@ -16,15 +16,35 @@ package version
 
 import (
 	"fmt"
+	"os"
 	"runtime"
+	"syscall"
+
+	"github.com/cilium/ebpf"
+	"kmesh.net/kmesh/daemon/options"
+	"kmesh.net/kmesh/pkg/logger"
+)
+
+const (
+	NewStart = iota
+	Restart
+	Update
```

Comment:
and update means what?

Reply:
For the future update scenario, if the two gitversions are different after the kmesh is restarted, it indicates that the update scenario is used and the previous configuration is not load.